src/Subheader/Subheader.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import muiThemeable from './../muiThemeable';
+import {mergeStyles, prepareStyles} from './../utils/styles';
+import Typography from '../styles/typography';
+
+const propTypes = {
+  /**
+   * Node that will be in subheader.
+   */
+  children: React.PropTypes.node,
+
+  /**
+   * If true, the subheader will be indented 72px
+   */
+  inset: React.PropTypes.bool,
+
+  /**
+   * Override the inline-styles of the root element.
+   */
+  style: React.PropTypes.object,
+};
+
+const defaultProps = {
+  inset: false,
+};
+
+let Subheader = ({children, muiTheme, inset, style, ...other}) => {
```

Comment:
That's a tough question :grin:. I would say `(props) =>` for when we add a new parameter in the future.